scripts/publish-docker.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# prepare and apply changeset
```

Comment:
Do these commands also create the tag and update the versions? If so, I would consider changing the name in "release.sh" or "bump-and-publish-docker.sh" or "tag-and-release-docker.sh", so that it is more obvious